static/css/main.css

```diff
@@ -5890,7 +5893,7 @@ figure {
     border-width: 0 15px;
     margin-right: -0.25em;
     vertical-align: top;
-    width: 50%;
+    width: 33.33333333%;
```

Comment:
Why do you need to remove it? Aren't the primary colors supposed to be three columns now?